modules/singleton-jdk/configure.sh

```diff
@@ -16,4 +16,7 @@ fi
 rpm -e --nodeps $(rpm -qa java-* | grep -v "^java-${JAVA_VERSION}-${JAVA_VENDOR}")
 
 # workaround for https://bugzilla.redhat.com/show_bug.cgi?id=2089626
```

Comment:
Lets reference the migrated JDK bug here: https://issues.redhat.com/browse/RHEL-3437